py/torch_tensorrt/dynamo/lowering/_decompositions.py

```diff
@@ -394,4 +394,5 @@ def get_decompositions(
         }
         return {**CORE_ATEN_DECOMPOSITIONS_FILTERED, **TORCH_TRT_DECOMPOSITIONS}
     else:
-        return {**ENABLED_TORCH_DECOMPOSITIONS, **TORCH_TRT_DECOMPOSITIONS}
+        # changes made here due to torch2.6 changes https://github.com/pytorch/pytorch/pull/135080
+        return {**_decomp_table_to_post_autograd_aten(), **TORCH_TRT_DECOMPOSITIONS}
```

Comment:
I think `ENABLED_TORCH_DECOMPOSITIONS` should still be kept, not replaced.

Reply:
@HolyWu  thanks for pointing out the issue
I am not very sure for the same operators which exists in both _decomp_table_to_post_autograd_aten() and ENABLED_TORCH_DECOMPOSITIONS, 
which one should override the other?
currently, I use the following sequence:
**ENABLED_TORCH_DECOMPOSITIONS,
**_decomp_table_to_post_autograd_aten(),
**TORCH_TRT_DECOMPOSITIONS,

Reply:
I think the sequence you use is fine because it matches the behavior in exported_program's run_decompositions().